yapapi/strategy.py

```diff
@@ -59,7 +181,9 @@ def __init__(
         self._activity = activity
 
     async def decorate_demand(self, demand: DemandBuilder) -> None:
-        """Ensure that the offer uses `PriceModel.LINEAR` price model."""
+        await super().decorate_demand(demand)
```

Comment:
Why do we need this now?
Does it have anything to do with the topic of this PR?
What is this for? `MarketStrategy` has empty `decorate_demand`, so this does nothing I guess?

Reply:
it does nothing but still, it would be expected for a strategy to do this if/when we change MarketStrategy - I believe it's a good pattern to suggest to anybody enabling their own strategies to follow

Reply:
But why would a base `MarketStrategy` have anything in `decorate_demand`?

This doesn't make any sense to me. So please, **let's keep code that makes sense now, not code that might one day maybe be useful in some abstract case we can't conceive now**.